integrations/sources/sql-server-cdc.mdx

```diff
@@ -87,7 +87,7 @@ CREATE TABLE [ IF NOT EXISTS ] table_name (
 WITH (
     snapshot='true'
 )
-FROM source TABLE table_name;
+FROM source TABLE database_name.table_name;
```

Comment:
Suggest to use a identifier `sqlserver_table_name` in the syntax, and add explaination in the table below.
```
sqlserver_table_name: identifier of SQL Server table in the format of 'database_name.schema_name.table_name'
```